src/templates/admin/elements/layout/key_value_above.html

```diff
@@ -0,0 +1,18 @@
+<div class="key-value-pair key-above">
+  <div class="key">
+    {{ key }}
+  </div>
+  <div class="value">
+    {% if list %}
+      <ul class="inline-list">
+        {% for item in value %}
+        <li>{{ item }}</li>
+        {% endfor %}
+      </ul>
+    {% elif render_line_breaks %}
+      {{ value|linebreaksbr|default_if_none:"No value supplied" }}
+    {% else %}
+      {{ value|default_if_none:"No value supplied" }}
+    {% endif %}
+  </div>
+</div>
```

Comment:
Can we use `default` instead of `default_if_none` to catch empty string values?

https://docs.djangoproject.com/en/5.0/ref/templates/builtins/#default